CONTRIBUTING.md

```diff
@@ -61,3 +61,14 @@ In an ideal world, every pull request is small, but the codebase is large and so
 [coc]: [https://github.com/openfga/openfga/CODE_OF_CONDUCT.md]
 [discord]: [https://discord.com/invite/pvbNmqC]
 
+## Ready to Contribute
+
+To contribute to the OpenFGA project, we ask that the changes are:
```

Comment:
Can we move these under pull request requirements?

* All contributions need to be [Apache-2.0](https://www.apache.org/licenses/LICENSE-2.0) licensed.
* Dependencies & sub dependencies have to have compatible licenses. The existing repositories have [FOSSA checks](https://fossa.com/) integrated as part of their GitHub Actions CI/CD workflows. New repositories will need to have similar checks in place.

* New contributions have to pass several security tests. The existing repositoris have [Snyk](https://snyk.io/) and [Semgrep](https://semgrep.dev/) integrated as part of their GitHub Actions CI/CD workflow. New repositories will need to have similar checks in place.

* Contributions should introduce minimal to no dependencies. This helps with maintenance as well as security and legal considerations.